lib/pure/browsers.nim

```diff
@@ -79,3 +111,36 @@ proc openDefaultBrowser*() {.since: (1, 1).} =
   ## .. code-block:: nim
   ##   block: openDefaultBrowser()
   openDefaultBrowserImpl("http:about:blank")  # See IETF RFC-6694 Section 3.
+
+proc findInstalledBrowsers*(limit = 3.Positive, followSymlinks = true): set[WebBrowsers] {.inline, since: (1, 3).} =
+  ## Try to find *all* installed web browsers, not just the default one.
```

Comment:
```suggestion
  ## Try to find *all* installed web browsers within `PATH`, not just the default one.
```

It's better to clarify this, since Windows and certain package managers (ie. `flatpak`) do not export their browsers to `PATH`. Also this may not work for macOS, which IIRC doesn't add installed applications to `PATH`.

So effectively this proc will only works on Linux...

Reply:
I agree, if you know how to improve it, please send your Pull Request to `os.findExe()`.
:)